autorest/azure/cli/token.go

```diff
@@ -63,7 +63,17 @@ func (t Token) ToADALToken() (converted adal.Token, err error) {
 
 // AccessTokensPath returns the path where access tokens are stored from the Azure CLI
 func AccessTokensPath() (string, error) {
-	return homedir.Expand("~/.azure/accessTokens.json")
+	// Azure-CLI allows user to customize the path of access tokens thorugh environment variable.
+	var accessTokenPath = os.Getenv("AZURE_ACCESS_TOKEN_FILE")
+	var err error
+
+	// Fallback logic to default path on non-cloud-shell environment.
+	// TODO(sushi): remove the dependency on hard-coding path.
```

Comment:
Instead of mentioning yourself, please open a new issue and mention it in this comment. :)